File: Assignment/C - CPP - Database/Module-4 OOPS Concept/practice/library.cpp
#include <iostream>
using namespace std;

class Library{
    public:
    int no_of_books = 0;
    string books[100];

    public:

    void addBook(string book){
        this->books[no_of_books] = book;
        no_of_books++;
        cout << "-> " + book + " has been added.\n";
    }

    void showBook(){
        for (int i = 0; i < no_of_books; i++)
        {
            cout << books[i] << "\n";   
        }
    }
};

int main(){
    
    Library l;

    l.addBook("Hello");
    l.addBook("World");

    l.showBook();
    

    return 0;
}

//switch case

File: Assignment/Python/Practice/library.py
class Library:
    def __init__(self):
        self.books = []
        self.no_of_books = 0
    
    def addBook(self, book):
        self.books.append(book)
        self.no_of_books = len(self.books)
        print(f"book {book} was added successfully.")
    
    def showBooks(self):
        print(f"The library has {self.no_of_books} books. The books are: ")
        if self.books:
            for i, book in enumerate(self.books):
                print(f"book no.{i+1}: {book}")
        else:
            print(f"No books available.")

l = Library()

l.addBook("book1")
l.addBook("book2")
l.addBook("book3")
l.addBook("book4")

l.showBooks()

    
    
